gozw.go

```diff
@@ -299,7 +300,7 @@ func (c *Client) FactoryReset() error {
 	return nil
 }
 
-func (c *Client) AddNode() (*Node, error) {
+func (c *Client) AddNode(progress chan PairingProgressUpdate) (*Node, error) {
```

Comment:
talked offline. Decided we could add AddNodeWithProgress instead of breaking existing contract.